crates/walrus-service/src/node.rs

```diff
@@ -171,7 +176,7 @@ pub mod config;
 pub(crate) mod consistency_check;
 pub mod contract_service;
 pub mod dbtool;
-pub mod events;
+pub mod event_blob_writer;
```

Comment:
Question: What is the plan with the event-blob writer? Is this going to stay with the node?